internal/pkg/fieldtags/testdata/src/tests/test.go

```diff
@@ -18,6 +18,7 @@ type Person struct {
 	password, creds      string      `levee:"source"`               // want "tagged field: password, creds"
 	secret               string      `json:"secret" levee:"source"` // want "tagged field: secret"
 	another              interface{} "levee:\"source\""             // want "tagged field: another"
+	hasCustomFieldTag    string      `example:"sensitive"`          // want "tagged field: hasCustomFieldTag"
```

Comment:
I'd like to also see a test that includes something like
```golang
`example:"sensitive,custom,additionaltag"`
```

Reply:
I think this is outside the scope of the current PR, but reasonable as a drive-by. Done.

Reply:
I dunno.  Implementation and the testing contract changed.  I think that puts it within the scope of this PR.  `</hair_splitting>`

Reply:
Fair enough. This should already have been tested anyway.